doc/doxygen/headers/reordering.h

```diff
@@ -73,16 +127,33 @@
  * consider flip-rotate or flip-orient-rotate as those cases are equivalent,
  * respectively, to the identity operation or the orientation = `true` case as
  * flip-rotate is equal to the identity operation. As a consequence, there are
- * only six valid orientations for triangles as faces of tetrahedra.
+ * only six valid orientations for triangles as faces of tetrahedra, pyramids,
+ * or wedges.
+ *
+ * Like the line case, the stored orientation value defines the way that the
+ * vertices of the face must be permuted to match the cell-local ordering. A
+ * consequence of this choice is that QProjector uses the inverse orientation
+ * (via ReferenceCell::get_inverse_combined_orientation()) to compute the
+ * locations of quadrature points, since either
```

Comment:
Let's make this a complete sentence:
```suggestion
 * locations of quadrature points, since either one of the following possibilities must be the case:
```